packages/os/os.spec

```diff
@@ -315,15 +315,41 @@ migrations=()
 for migration in $(find %{_builddir}/sources/api/migration/migrations/v[0-9]* -mindepth 1 -maxdepth 1 -type d); do
     migrations+=("-p $(basename ${migration})")
 done
+
 # Store the output so we can print it after waiting for the backgrounded job.
+exec 3>&1 4>&2
 static_output="$(mktemp)"
+exec 1>"${static_output}" 2>&1
 # Build static binaries in the background.
 %cargo_build_static --manifest-path %{_builddir}/sources/Cargo.toml \
     -p apiclient \
     ${migrations[*]} \
-    >> ${static_output} 2>&1 &
+    &
 # Save the PID so we can wait for it later.
 static_pid="$!"
+exec 1>&3 2>&4
+
+%if %{with aws_platform} || %{with aws_k8s_family}
```

Comment:
Is this one of those few exceptions where it is OK to make the spec file platform or family-aware?

Reply:
> Is this one of those few exceptions where it is OK to make the spec file platform or family-aware?

This is just to keep the existing conditional compile logic in effect.

When these crates don't need to be conditionally compiled any more, we can just drop the conditions and always do the AWS SDK consumer crates in a separate pass (like we do with static builds).